### ISSUE TITLE
User login bug

### ISSUE THREAD

**Issue by wanclem** [magnetsystems/message-sdk-java-android#60]
Examine the code below,even when I set remember me to true,am still forced to login over and over and over again anytime the app starts because User.getCurrentUser() always returns null.

User.login(userName, password, true, new ApiCallback<Boolean>(){});
Please fix in android sdk.

**Comment by jimxj** [comment]
Hi @WanClem , there is a bug for rememberMe before 2.5.0. Can you update your version to 2.5.0 and try again ?

To use rememberMe, you need to check the status like following : 

```
if (User.SessionStatus.LoggedIn == User.getSessionStatus()) {
            goToHomeActivity(); 
        } else if (User.SessionStatus.CanResume == User.getSessionStatus()) {
            User.resumeSession(new ApiCallback<Boolean>() {
                @Override 
                public void success(Boolean aBoolean) {
                    if (aBoolean) {
                        goToHomeActivity(); 
                    } else { 
                        handleError(""); 
                    } 
                } 


                @Override 
                public void failure(ApiError apiError) {
                    handleError(apiError.getMessage());
                } 


                private void handleError(String errorMessage) {
                    Logger.debug(TAG, "Failed to resume session due to ", errorMessage);
                    goToLoginActivity(); 
                } 
            }); 
        } else { 
            goToLoginActivity(); 
        } 
```

You can check out our new sample app which demonstrates the API usage : https://github.com/magnetsystems/message-samples-android/tree/develop

Specially this class : https://github.com/magnetsystems/message-samples-android/blob/develop/Messenger/app/src/main/java/com/magnet/magnetchat/ui/activities/sections/splash/SplashActivity.java

**Comment by wanclem** [comment]
Wow! Then I guess the bug has not yet being fixed coz am using version 2.5.0

**Comment by jimxj** [comment]
Let me know how it goes.
